cmd/storage-class.go

```diff
@@ -48,6 +52,12 @@ type storageClassConfig struct {
 	RRS      string `json:"rrs"`
 }
 
+// Validate if storage class in metadata
+// Only Standard and RRS Storage classes are supported
+func isValidStorageClassMeta(sc string) bool {
+	return sc == reducedRedundancyStorageClass || sc == standardStorageClass || sc == ""
```

Comment:
wait are we saying that "x-amz-storage-class" header if set with empty string is good with AWS S3? have you tested this? @nitisht

Reply:
good catch, S3 doesn't allow empty string. I have changed the behaviour here accordingly.